modules/common-workflows/pages/workflow-install-from-raw-image.adoc

```diff
@@ -1,62 +1,61 @@
 
-{productname} offers various image formats, including a raw format. This guide demonstrates how to set up {productname} Server using the raw image.
 
 
 == Use Case
 
-RAW images are pre-built, ready-to-use representations of a running operating system. Instead of being installed through a traditional installer, they are directly copied to the hard disk of the target host.
-Raw image format is flexible and compatible with a wide range of virtualization and deployment tools. This guide provides essential information about these pre-configured images.
+RAW images are pre-built, ready-to-use representations of a running operating system. 
+Instead of being installed through a traditional installer, they are directly copied to the hard disk of the target host.
+RAW image format is flexible and compatible with a wide range of virtualization and deployment tools. 
+
+This guide provides essential information about these pre-configured images.
+
 
 
 == Outcome 
 
 Successful completion of this workflow results in successful installation of {productname} Server.
 
 
-== Step-by-step Preparation Instructions
 
+== Step-by-step Instructions
 
 [WARNING]
 ====
 This workflow is complex and time-consuming.
 Make sure that adequate testing is done before deploying the procedure in live environment.
 ====
 
-.Procedure: Prerequisites
+.Procedure: Prepare the Prerequisites
 [role=procedure]
 . *Download the Image:* Download the architecture-specific [literal]``.raw.xz`` image of SUSE Manager 5.0 Server for your target block size.
-
-
-. *Make Image Accessible:* Copy the image to a web server (preferred), NFS server, or expand the .raw file to add directly as a disk in a virtual environment (if supported).
-
-
-. *Boot Media:* Provide a bootable ISO ({sles} 15 SPX or {sle-micro} 5.5) to access the "Rescue System." Add netsetup=1 on the kernel command line for networking, or configure IP via ifcfg.
-
+. *Make Image Accessible:* Copy the image to a web server (preferred), NFS server, or expand the [literal]``.raw`` file to add directly as a disk in a virtual environment (if supported).
+. *Boot Media:* Provide a bootable ISO ({sles} 15 SPX or {sle-micro} 5.5) to access the "Rescue System." 
+  Add [literal]``netsetup=1`` on the kernel command line for networking, or configure IP via [command]``ifcfg``.
 . *Target Disk Setup:* In VMware, create:
 .. A 40GB root disk
 .. Additional storage for persistent SUSE Manager data
+
 											
 .Procedure: Installation Steps
-										
+[role=procedure]										
 . *Prepare Network and Boot Environment*
-
 .. Create a new instance, attach the root and persistent storage, then attach the {sles} or {sle-micro} ISO as a virtual CD.
-.. Boot the instance to the ISO, selecting the “Rescue System.”
+.. Boot the instance to the ISO, selecting the [literal]``Rescue System``.
 .. Add the following on the command line to enable network configuration.
-
 +
 ----
 netsetup=1
```

Comment:
This may work now, but AFAIK SLE16 (and I wonder if SL-Micro 6.1) does not ship linuxrc.

How about using `ip=` from dracut ( https://manpages.opensuse.org/Tumbleweed/dracut/dracut.cmdline.7.en.html#Network )

e.g. `ip=dhcp` for automatic assignment.

Reply:
We are talking about SUSE Manager 5.0 here, and are recommending people boot from SLE15/SLE Micro 5.X  generation rescue media.  The dracut command does not work from the linuxrc offered there.
We are not supporting SL Micro 6.1 nor SLE 16 media for these purposes until a later release of SUSE Manager.  

Quite often customers may not have DHCP available.  By choosing 'netsetup=1' they are prompted to either choose DHCP or configure manual settings.    Alternatively  they can follow the linuxrc 'ifcfg' configuration referenced here:
https://en.opensuse.org/SDB:Linuxrc#Using_ifcfg

Reply:
Well of course, I was hoping to have something which will work later as well. Still we can recommend SLE15 boot media till 2037, so I guess we are good.